src/Compilers/VisualBasic/Portable/Lowering/LocalRewriter/LocalRewriter_InterpolatedString.vb

```diff
@@ -50,7 +50,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
         Private Function RewriteInterpolatedStringConversion(conversion As BoundConversion) As BoundExpression
 
-            Debug.Assert(conversion.ConversionKind = ConversionKind.InterpolatedString)
+            Debug.Assert((conversion.ConversionKind And ConversionKind.InterpolatedString) = ConversionKind.InterpolatedString)
```

Comment:
That would be incorrect - ConversionKind.InterpolatedString is a combination of bits, we are checking all of them. I believe other instances are of the same nature.